cpp/cograph_recognition/cograph/Recognition.cpp

```diff
@@ -0,0 +1,549 @@
+#include <cograph_recognition/CographRecognition.hpp>
+#include <list>
+#include <set>
+#include <bits/stdc++.h>
+#include <graph/GraphTools.hpp>
+using namespace std;
```

Comment:
Please remove it, in general I avoid polluting local namespace.

Reply:
fixed